guides/v2.1/javascript-dev-guide/widgets/widget_modal.md

```diff
@@ -202,6 +209,36 @@ Called when the modal window is opened.
 - the TAB key: set focus to the next focusable element (looped inside the modal window)
 - the SHIFT+TAB keys combination: set focus to the previous focusable element (looped inside the modal window)
 
+## Code sample
+
+The following example shows how to initialize the modal widget and pass options during the initialization.
+
+```html
+<button type="button" class="action" data-trigger="trigger">
+    <span data-bind="i18n: 'Click Here'"></span>
+</button>
+<div data-bind="mageInit: {
+        'Magento_Ui/js/modal/modal':{
+            'type': 'popup',
+            'title': 'Popup title',
+            'trigger': '[data-trigger=trigger]',
+            'responsive': true,
+            'buttons': [{
+                text: 'Submit',
+                class: 'action'
+            }]
+        }}">
+    <div class="content">
+        Popup Content
+    </div>
+</div>
+```
+
+### Result
+
+The result is a modal and a button (_Click Here_) which opens the modal.
+
+![Tabs Widget]({{ page.baseurl }}/javascript-dev-guide/widgets/images/modal-widget-result.png)
```

Comment:
Please, double check the `[Tabs Widget]` title here. The picture is about a modal widget example.